src/components/query-viewer-card/controller/card-utils.ts

```diff
@@ -302,13 +306,15 @@ const showDataAsMapping: {
 // @todo Complete the implementation of this function by comparing it to the Web implementation
 export function valueOrPercentage(
```

Comment:
`[valueOrPercentage]` It would be nice to add some tests using this utility with different values to validates the new behavior.